### PR TITLE
Add cmov* support to x86.nz

### PR DESCRIPTION
**Pull request by Svenito**

### CODE REVIEW

File: libr/asm/p/asm_x86_nz.c [hunk 1]
```diff
@@ -517,6 +517,134 @@ static int opcall(RAsm *a, ut8 *data, const Opcode op) {
...
+	int mod_byte = 0;
+	int offset = 0;
+
+	if (op.operands[0].type & OT_MEMORY ||
+	    op.operands[1].type & OT_CONSTANT) {
+		return -1;
+	}
+
+	data[l++] = 0x0f;
+	if (!strcmp (op.mnemonic, "cmovo")) {
```
Comment (radare) [inline libr/asm/p/asm_x86_nz.c]: what about comparing "cmov" first, and then checking only the suffix?

Comment (radare) [inline libr/asm/p/asm_x86_nz.c]: you can even do that in a switch char by char

**Comment by radare** [comment]
lgtm after this simplification

File: libr/asm/p/asm_x86_nz.c (hunk 1 above)
Comment (Svenito) [inline libr/asm/p/asm_x86_nz.c]: With a switch char by char I wouldn't be able to group instructions that have the same opcode.
I'd prefer checking the suffixes only

**Pull request merged**